webapp/src/ts/services/enketo.service.ts

```diff
@@ -825,7 +825,7 @@ export class EnketoService {
 
 interface ContactSummary {
   id: string;
-  xmlStr: string;
+  xmlStr?: string;
```

Comment:
```suggestion
  xml: Document;
```

I think we just need to fix the name of this field.  AFICT, `xmlStr` is not ever used....

Reply:
That's correct! Done